python-sdk/src/astro/lineage/extractor.py

```diff
@@ -37,6 +37,7 @@ def get_operator_classnames(cls) -> list[str]:
             "LoadFileOperator",
             "MergeOperator",
             "TransformOperator",
+            "ExportFileOperator",
```

Comment:
nit: Look like the list is sorted here, can we keep it in the same way

Reply:
changed